src/treq/test/test_treq_integration.py

```diff
@@ -241,6 +244,150 @@ def test_failed_basic_auth(self):
         self.assertEqual(response.code, 401)
         yield print_response(response)
 
+    @inlineCallbacks
+    def test_digest_auth(self):
+        """
+            Test successful Digest authentication
```

Comment:
It would be nice to have test docstrings that follow the rules in https://jml.io/pages/test-docstrings.html .

Reply:
Fixed the indentation issue.